editors/cedet/Portfile

```diff
@@ -48,8 +48,8 @@ destroot {
         global  destroot worksrcpath
         upvar   lispdir lispdir
 
-        xinstall -d $destroot$lispdir/$package_dir
-        foreach f [glob -directory $worksrcpath/$package_dir *] {
+        xinstall -d ${destroot}$lispdir/$package_dir
+        foreach f [glob -directory ${worksrcpath}/$package_dir *] {
```

Comment:
Surprised you didn't do "lispdir" and "package_dir" while you were here. (See also the rest of this file.)

Reply:
Thanks. I did a global search and replace, which is also why I missed the quoted `$prefix`. Both fixed.